db/fixtures/server_roles.csv

```diff
@@ -1,5 +1,6 @@
 name,description,max_concurrent,external_failover,role_scope
 automate,Automation Engine,0,false,region
+cockpit_ws,Cockpit,1,false,region
```

Comment:
Can you explain what the `ws` means?  I really don't like abbreviations and would just prefer that everywhere you have WS it was spelled out.  However I do understand that you are also trying to align with the executable name from `/usr/libexec/cockpit-ws`, so perhaps in some cases it does make sense.  If I knew what it meant I could probably help decide where in the code the abbreviation does or doesn't make sense.